.tekton/monitoring-console-plugin-0-5-1-2-pull-request.yaml

```
@@ -0,0 +1,627 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/rhobs/konflux-coo?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "true"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: |-
+      event == "pull_request" && target_branch == "release-1.2" &&
+              (".tekton/monitoring-console-plugin-0-5-1-2-pull-request.yaml".pathChanged() ||
+              ".tekton/monitoring-console-plugin-0-5-1-2-push.yaml".pathChanged() ||
+              "Dockerfile.ui-monitoring".pathChanged() ||
+              "ui-monitoring/***".pathChanged())
```

This pattern match has changed to 
```suggestion
              "ui-monitoring".pathChanged())
```
did you run the latest version of the script? It also might be that the latest script does not have this fix

---

lol is there a script to generate these yamls? I manually created these, I'll add the pattern match `***` in the next push.

---

Change is pushed; I manually removed the `***`.